base/setup/usetup/consup.h

```diff
@@ -117,6 +117,12 @@ CONSOLE_PrintTextXYN(
 	IN SHORT len,
 	IN LPCSTR fmt, ...);
 
+VOID
+CONSOLE_ClearTextXY(
```

Comment:
I'd argue calling the already public function `CONSOLE_ClearStyledText` might be cleaner.

Reply:
@ThFabba, so your suggestion would be a single line addition something like the following?

**CONSOLE_ClearStyledText(6, 10, TEXT_TYPE_REGULAR, 74);**